docs/user-guide-policy-configure-bond-interface.md

```diff
@@ -0,0 +1,70 @@
+# Tutorial: Create a Bond Interface and Connect it to a Node Interface
+
+Use Node Network Configuration Policy to configure a new bond interface `bond0`
+with slaves `eth1` and `eth2`.
+
+## Requirements
+
+Before we start, please make sure that you have your Kubernetes/OpenShift
+cluster ready. In order to do that, you can follow the guides of deployment on
+[local cluster](deployment-local-cluster.md) or your
+[arbitrary cluster](deployment-arbitrary-cluster.md).
+
+## Configure bond
+
+All you have to do in order to create the bond on all nodes across cluster is
+to apply the following policy:
+
+```yaml
+cat <<EOF | ./kubevirtci/cluster-up/kubectl.sh create -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond0-eth1-eth2-policy
+spec:
+  desiredState:
+    interfaces:
+    - name: eth2
```

Comment:
@EdDev do we need to change the interface ipv4 to false when we make them slaves to the bond or we can remove that?

Reply:
I think we don't need to mention them at all. If there are slaves/ports in bond/bridge, it creates slave connections to respective ifaces. This is redundant.

Reply:
Please see https://github.com/nmstate/kubernetes-nmstate/pull/159